docs/custom-metric-filter.md

```diff
@@ -0,0 +1,151 @@
+# ACM Custom Metric Filtering
+
+TL;DR: This guide explains how to adjust the custom metrics that Config Sync
+exports to Prometheus, Cloud Monitoring (formerly known as Stackdriver), and
+Monarch (Google's internal metrics aggregator).
+
+## Background
+
+When Config Sync is installed on a GKE cluster where a
+[default service account](http://cloud/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+is available, or on a GKE cluster that has
+[Workload Identity](http://cloud/kubernetes-engine/docs/how-to/workload-identity)
+enabled and
+[IAM is correctly setup](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#custom-monitoring),
+Config Sync will try to apply the Open Telemetry Collector ConfigMap that contains
+[googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter)
+to write metrics into Cloud Monitoring. The default configuration contains a
+bare minimum list of metrics, but some customers might want to configure
+different backends or fine tune the metric list for better monitoring solutions.
+
```

Comment:
Add a bit on the difference between the 3 different configmaps with similar names, maybe bullets and a paragraph. Explain what they do and that to ensure your modifications aren't reverted, you need to use the -custom one.

Reply:
Added a paragraph and linked to the public doc so that we don't repeat ourselves too many times.